README.md

```
@@ -7,17 +7,45 @@
+    (b) Clone the Repo: `git clone https://github.com/AOSSIE-Org/Resonate-Backend`
 
-2. Follow [this guide](https://docs.livekit.io/cloud/project-management/keys-and-tokens/) to obtain your `HOST`, `API-KEY`, `API-SECRET` from [livekit-cloud](https://livekit.io/cloud).
+2. Set up environment variables
 
-3. Create a new project on [Appwrite Cloud](https://appwrite.io/) or self host it locally by pulling their [docker image](https://appwrite.io/docs/self-hosting). Know more about Appwrite [here](https://appwrite.io/docs).
+```bash
+cp .env.example .env
+```
+
+Then fill in the values in `.env`:
+   - Leave the database/collection IDs as they are (they match the project structure)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)